src/components/InvertedFlatList/CellRendererComponent/index.android.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../../styles/styles';
+
+const propTypes = {
+    /** Position index of the list item in a list view */
+    index: PropTypes.number.isRequired,
+};
+
+function CellRendererComponent(props) {
+    return (
+        <View
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...props}
```

Comment:
I'm curious, what's the alternative way of doing this, and don't we do it here instead of suppressing the lint rule?

Reply:
I am not sure what is an alternative way. React passes bunch of props to the `CellRendererComponent`. 
https://github.com/facebook/react-native/blob/e22bd7f6a969321607246e64c9b4e96ef9265fb1/packages/virtualized-lists/Lists/VirtualizedList.d.ts#L90-L98

Theoretically, I could specify all of them in prop types and pass them as separate props in the `CellRendererComponent`. This is a bad idea:
1. We care only about props that we are working with and not some internal FlatList props.
2. If props list changes, on the React side and we will have to touch the component again.

I think we are better off no defining props that are irrelevant to us.

Reply:
That makes sense. Do you know why we have the `jsx-props-no-spreading` es lint rule then? I'm wondering if we should remove it, but this isn't a blocker as is not something that we should do in this PR